apps/editors/templates/editors/review.html

```diff
@@ -163,24 +163,20 @@ <h3 id="history">
       </div>
 
       <div class="review-actions-section review-actions-files data-toggle"
-           data-value="{{ actions_minimal|join("|") }}"{% if allow_unchecking_files %} data-uncheckable="1"{% endif %}>
+           data-value="{{ actions_minimal|join("|") }}">
         <label for="id_addon_files"><strong>{{ form.addon_files.label }}</strong></label>
         <ul>
             {% for pk, label in form.fields.get('addon_files').choices %}
             <li>
               <label for="file-{{ pk }}"{% if pk in form.addon_files_disabled %} class="light"{% endif %}>
                 <input id="file-{{ pk }}" type="checkbox" value="{{ pk }}" name="addon_files"
-                       {% if pk in form.addon_files_disabled %}disabled=""{% endif %} />
+                       disabled="disabled" checked="checked" />
```

Comment:
The same for `checked` I believe, I was used to writing it the "old way", but i'll modify, thanks.